app/main.py

```diff
@@ -16,4 +16,27 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+def sorted_variables(values):
+    mutable_types = (list, set, dict)
+    immutable_types = (int, float, str, tuple, bool)
+    sorted_values = {"mutable": [], "immutable": []}
+    for i in values:
+        if isinstance(i, mutable_types):
+            sorted_values["mutable"].append(i)
+        elif isinstance(i, immutable_types):
+            sorted_values["immutable"].append(i)
+    return sorted_values
+
+all_values = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins,
+]
+sorted_variables = sorted_variables(all_values)
```

Comment:
You are reassigning the `sorted_variables` function to the result of the function call. This will overwrite the function definition, causing an error if you try to call `sorted_variables` again. Consider using a different variable name for the result, such as `sorted_result`.